File: lexilogicclient/src/main/java/org/amalgam/backend/referenceobjects/Lobby.java
package org.amalgam.backend.referenceobjects;

public class Lobby {
    private int lobbyID;
    private String winner;

    public Lobby(int lobbyID, String createdBy, String winner) {
        this.lobbyID = lobbyID;
        this.winner = winner;
    }

    public Lobby() {
        this.lobbyID = 0;
        this.winner = null;
    }

    public int getLobbyID() {
        return lobbyID;
    }

    public void setLobbyID(int lobbyID) {
        this.lobbyID = lobbyID;
    }

    public String getWinner() {
        return winner;
    }

    public void setWinner(String winner) {
        this.winner = winner;
    }
}

File: Client_Python/Utilities/Reference_Objects/Lobby.py
class Lobby:
    def __init__(self, lobby_id=None, winner=None, winner_score=None):
        self._lobby_id = lobby_id
        self._winner = winner
        self._winner_score = winner_score

    # Getter
    def get_lobby_id(self):
        return self._lobby_id

    def get_winner(self):
        return self._winner

    # Setter
    def set_lobby_id(self, value):
        self._lobby_id = value

    def set_winner(self, value):
        self._winner = value

    def set_winner_score(self, value):
        self._winner_score = value

    def get_winner_score(self):
        return self._winner_score
